core/tests/test_utils.py

```diff
@@ -249,7 +249,7 @@ class TestBase(unittest.TestCase):
         }
     }
 
-    VERSION_27_STATE_DICT = {
+    VERSION_32_STATE_DICT = {
```

Comment:
@seanlip is this an okay change?

Reply:
Probably not. @rohitkatlaa please make sure to read the docstrings and code for the functions you are modifying.

Reply:
@seanlip I am sorry for not noticing it before. I think that the dictionary is used to load older version of the state because this is used to test whether the schema conversion works correctly.